addons/payment_authorize/models/payment_transaction.py

```diff
@@ -95,18 +95,37 @@ def _send_refund_request(self, amount_to_refund=None, create_refund_transaction=
                 create_refund_transaction=create_refund_transaction,
             )
 
-        refund_tx = super()._send_refund_request(
-            amount_to_refund=amount_to_refund, create_refund_transaction=False
-        )
+        # Only full refund is possible.
```

Comment:
I feel like this is useless, since the `support_refund` is set as `full_only`

Reply:
I mean the comment (didn't expect the review to show it like this >.<)

Reply:
done :)